src/components/Calculator.js

```diff
@@ -1,63 +1,87 @@
 import React from 'react';
+import calculate from '../logic/calculate';
+import operate from '../logic/operate';
 
 class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      calculateObject: {
+        total: null,
+        next: null,
+        operation: null,
+
+      },
+    };
+  }
+
```

Comment:
Hey @abdallahmalima

The objective of this milestone was to refactor the Calculator component from a class-based component to a functional component using [hooks](https://reactjs.org/docs/hooks-intro.html). Your Calculator component is still class-based.

Here's an example of how it should look like:
`// ./src/components/Calculator.js`
```
import React, { useState } from 'react';

const Calculator = () => {
     const [state, setState] = useState(
     {
        total: null,
        next: null,
        operation: null,
      });

     handleClick=(e) => {
      //Update state using setState
     }

     return (
      <div>
       // Calculator app design goes here
      </div>
     )
};

export default Calculator; 
```

Reply:
well resolved! ✔️